modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs

```diff
@@ -67,10 +67,12 @@ public void TestLocalMetrics()
             var remoteMetrics = metrics.Item2;
 
             Assert.AreEqual(1, localMetrics.Size);
+            Assert.AreEqual(1, localMetrics.SizeLong);
```

Comment:
You need to add your metrics to CacheMetricsTest.TestMetricsPropagation test. And do it in java tests as well.

Reply:
Done